src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RoleChangeRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.gelecekbilimde.scienceplatform.auth.model.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Sadece Getter Setter yeterli
```suggestion
@Getter
@Setter
```